kartograf/merge.py

```diff
@@ -35,30 +35,47 @@ def merge_pfx2as(context):
     general_merge(base_file, rv_file, rv_filtered_file, out_file, context.args.silent, context.args.workers)
     shutil.copy2(out_file, context.final_result_file)
 
+def seed_base_dict(base_dict):
```

Comment:
for ipv4, easy to preseed the map with possible values. Doesn't work for ipv6 though.